package.json

```diff
@@ -43,39 +42,26 @@
     "@types/node": "^9.4.6",
     "@types/react": "^16.0.38",
     "@types/react-dom": "^16.0.4",
-    "@types/react-router": "^3.0.15",
     "@types/redux-mock-store": "0.0.13",
-    "@types/webpack-env": "^1.13.5",
-    "awesome-typescript-loader": "^3.4.1",
     "babel-core": "^6.26.0",
     "babel-preset-env": "^1.6.1",
     "cross-env": "^5.1.3",
-    "css-loader": "^0.28.9",
     "deep-freeze": "0.0.1",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "enzyme-to-json": "^3.3.1",
-    "extract-text-webpack-plugin": "^3.0.2",
-    "file-loader": "^1.1.6",
-    "foreman": "^2.0.0",
-    "html-webpack-plugin": "^2.30.1",
+    "foreman": "^3.0.1",
     "identity-obj-proxy": "^3.0.0",
     "if-env": "^1.0.4",
     "jest": "^22.4.2",
     "node-sass": "^4.7.2",
     "raf": "^3.4.0",
     "redux-mock-store": "1.4.0",
     "rimraf": "^2.6.2",
-    "sass-loader": "^6.0.6",
-    "style-loader": "^0.20.2",
     "ts-jest": "^22.4.1",
     "tslint": "^5.9.1",
     "tslint-react": "^3.5.1",
-    "typescript": "^2.7.2",
-    "url-loader": "^0.6.2",
-    "webpack": "^3.11.0",
-    "webpack-dev-server": "^2.11.1",
-    "webpack-merge": "^4.1.1"
+    "typescript": "^2.7.2"
   },
   "jest": {
     "testRegex": "\\.spec\\.tsx?$",
```

Comment:
Add a setting to jest section to make test work again:
```
"testURL": "http://localhost/",
```
more info: https://github.com/jsdom/jsdom/issues/2304